django/chat/models.py

```diff
@@ -148,6 +159,7 @@ class ChatOptions(models.Model):
     chat_model = models.CharField(max_length=255, default="gpt-4o")
     chat_temperature = models.FloatField(default=0.1)
     chat_system_prompt = models.TextField(blank=True)
+    chat_agent = models.BooleanField(default=True)
```

Comment:
Agent is on by default in chat mode, to better redirect questions about e.g. HR/pay to the corporate library.
This is heavily dependent on Library names and descriptions (and data source names) being sufficiently descriptive.